dojo_theme/templates/components/navbar.html

```diff
@@ -15,6 +15,90 @@
     </button>
     <div class="collapse navbar-collapse" id="base-navbars">
       <ul class="navbar-nav mr-auto">
+        <li id="challengeMenuButton" class="d-none nav-item dropdown">
```

Comment:
What do you think about moving this to the right hand side, and making it always-condensed (like the logout button?). On a note-to-self, we should also always-condense the Profile and Settings probably, to keep the navbar from completely filling up. As things are right now, that navbar is super cluttered.

Reply:
This might be a cool result:
![image](https://github.com/user-attachments/assets/e340f173-5ab1-4edf-9e50-2464d1999091)

Reply:
Yeah I also thought about the flag symbol only.
As it might (hopefully) become a tool that people use more often then the dojo link I would rather put it to the left even without the label.
But I'm also fine with it being on the right side.

I really like that little panel 😁

Reply:
![grafik](https://github.com/user-attachments/assets/686167a8-119c-4f5e-a9b2-db13a105e42e)
this is result now